data/campaigns/Winds_of_Fate/story/Morogor_Dialect.txt

```diff
@@ -94,11 +94,11 @@ A race of lesser dragons. They are bipedal, diurnal and have a high capacity for
 
 - Drakish -
 The tongue of the Drakes of Morogor.
-It is a very archaic dialect of the common tongue spoken by most races, making communication with others difficult.
+As its structure is alien compared to the tongues spoken by other races, communication between the races can be difficult.
 
 - (Exonym) Caster -
 Also called Hurler.
-Ancient word for human, referring to the human ability to throw missile weapons, which is impossible for drakes due to their inferior arm articulation.
+Ancient word for human, referring to the human propensity for throwing missile weapons.
```

Comment:
Let's at least leave a suggestion that human anatomy is much better at throwing missiles powerfully and accurately than drake anatomy is. Drakes already have enough impressive physical features, like flight, fire breathing, tough scales. and size. This gives drakes something oddly specific to appreciate about the human species, in return,

Reply:
I'd prefer not to give an explanation for _why_ the drakes don't use missiles while humans do, to allow others who care to fill in the explanation in their own addon.